packages/node_modules/@node-red/nodes/core/function/10-function.html

```diff
@@ -71,12 +71,13 @@
             });
 
             RED.library.create({
-                url:"functions", // where to get the data from
-                type:"function", // the type of object the library is for
-                editor:this.editor, // the field name the main text body goes to
-                mode:"ace/mode/nrjavascript",
-                fields:['name','outputs'],
-                ext:"js"
+                url: "functions", // where to get the data from
+                type: "function", // the type of object the library is for
+                editor: this.editor, // the field name the main text body goes to
+                nodeInfoEditor: this.nodeInfoEditor,
```

Comment:
Whilst reviewing #2498 I added this comment which I think applies the same here: https://github.com/node-red/node-red/pull/2498#discussion_r401714760

In summary (but please read the full comment), I don't think we should specify `nodeInfoEditor` here. Instead the `fields` property should also accept objects that identify the property and provide custom getter and setter functions.

On that basis, we should co-ordinate this PR with @HiroyasuNishiyama and agree if we are going to make the change I propose, and then which of your PRs we use to do it (or if I should implement it and then these PRs can be updated to use it...).

Reply:
With #2498 now merged, we can modify this code to avoid hardcoding in the info field.

I'll make the necessary changes as part of the merge.

Reply:
Thank you for your consideration. :)